cli/src/test-utils/test-logger.ts

```diff
@@ -0,0 +1,36 @@
+import chalk from "chalk";
+
+const TestLogger = {
+  error,
+  log,
+  mute,
+  success,
+  warn
+};
+
+export default TestLogger;
+
+function error(message: string) {
+  // tslint:disable-next-line: no-console
+  console.log(chalk.red(message));
```

Comment:
Should this use `console.error` so it goes onto `process.stderr` instead of `process.stdout`? same thing with `warn()`. Or does it make more sense to use a single stream?

Reply:
I'm impartial to this. Is there a tangible benefit?

Reply:
It can be useful if you need to distinguish between expected output of a CLI tool (to pipe it to another tool) vs error messages, but I don't think that's the case here. Leave it as is.